initramfs/scripts/ubuntu-core-rootfs

```diff
@@ -132,6 +132,23 @@ handle_writable_paths()
 			continue
 		fi
 	done
+
+        # now apply the defaults on top
+        handle_writable_defaults
+}
+
+# handle_writable_default will apply system defaults *once* after
+# the handle_writable_path got populated
+handle_writable_defaults()
+{
+        srcpath="${rootmnt}/writable/system-data/_writable_defaults"
+        if [ ! -d "$srcpath" ] || [ -e "${srcpath}/.done" ]; then
+                return
+        fi
+
+        dstpath="${rootmnt}/writable/system-data"
+        cp -a "$srcpath/*" "$dstpath"
```

Comment:
It seems that quoting is messing things up... Took me a while to figure why this didn't have an effect in the complete spread test despite of .done being there...

Compare:
```
user1@localhost:~$ cp -a "/writable/system-data/_writable_defaults/*" "/tmp"
cp: cannot stat '/writable/system-data/_writable_defaults/*': No such file or directory
```

But this works:
```
user1@localhost:~$ cp -a /writable/system-data/_writable_defaults/* "/tmp"
```

It would be nice if a failure was visible somewhere, or at the very least .done was not created on error (but not super critical and maybe not too useful really).

Reply:
Nice catch, shell and I will not become friends anytime soon. I added a "proper" test now.